sss/tests/test_geoid.py

```diff
@@ -5,13 +5,52 @@
 import pandas as pd
 import pytest
 
+
 from sss.data import DATA_PATH
 from sss.geoid import GeoID, geo_identifier_creator
 
 
 @pytest.mark.filterwarnings("ignore:Unknown extension is not supported and will be")
-def testgeo_identifier_creator():
+def testgeo_identifier_creator(capsys):
     """Test to see if county_table and cpi_table are joined together."""
+
+    # tests print message is created when we have duplicate rows in CPI table
+    geo_identifier_creator(
+        os.path.join(DATA_PATH, "geoid_data", "SSScounty-place-list_20220720.xlsx"),
+        os.path.join(
+            DATA_PATH,
+            "geoid_data",
+            "StateAbbreviation_Regions_07192022_AKu_duplicate_rows.xlsx",
+        ),
+    )
+
+    out, _ = capsys.readouterr()
+    assert (
+        "Merge sucessful, but new rows were created due to \
+                    duplications in right(CPI) table"
+        in out
+    )
+
+    # test error message
+    with pytest.raises(Exception) as exc_info:
+        geo_identifier_creator(
+            os.path.join(
+                DATA_PATH,
+                "geoid_data",
+                "SSScounty-place-list_20220720_misnamed_column.xlsx",
+            ),
+            os.path.join(
+                DATA_PATH, "geoid_data", "StateAbbreviation_Regions_07192022_AKu.xlsx"
+            ),
+        )
+
+    assert exc_info.type == ValueError
+    assert exc_info.value.args[0] == (
+        "Cannot merge, please check the two columns"
+        + " are named as 'state_alpha' and 'USPS Abbreviation'"
+    )
```

Comment:
Same comment here. You can do all this with the `pytest.raises` call, just pass the expected message to the `match` parameter in `pytest.raises`, so something like this:

```
with pytest.raises(
    ValueError,
    match="Cannot merge, please check the two columns"
    " are named as 'state_alpha' and 'USPS Abbreviation'",
):
```
Note that you also do not need the `+` between the strings because python concatenates strings on separate lines.